client/signup/steps/theme-selection/index.jsx

```diff
@@ -58,14 +92,38 @@ module.exports = React.createClass( {
 		);
 	},
 
+	renderThemePreview() {
+		if ( this.state.previewTheme ) {
+			return (
+				<ThemePreview
+					showPreview={ true }
+					showExternal={ false }
+					theme={ this.state.previewTheme }
+					buttonLabel={ this.translate( 'Pick this Theme' ) }
```

Comment:
Also see #6952. One of these will need a rebase when the other is merged to update these prop names.